File: Python/Manusia.py
class Manusia:
    __nama = ""
    __nohp = ""

    def __init__(self, nama="", nohp=""):
        self.__nama = nama
        self.__nohp = nohp
    
    def getNama(self):
        return self.__nama
    
    def setNama(self, nama):
        self.__nama = nama
    
    def getnohp(self):
        return self.__nohp
    
    def setnohp(self, nohp):
        self.__nohp = nohp

File: CPP/Manusia.cpp
#include <bits/stdc++.h>
#pragma once
using namespace std;

class Manusia {
private:
    string nama;
    string nohp;

public:
    Manusia(string nama = "", string nohp = "") {
        this->nama = nama;
        this->nohp = nohp;
    }

    string getNama() {
        return nama;
    }

    void setNama(string nama) {
        this->nama = nama;
    }

    string getnohp() {
        return nohp;
    }

    void setnohp(string nohp) {
        this->nohp = nohp;
    }
};
